fineract-provider/src/test/java/org/apache/fineract/batch/command/CommandStrategyProviderTest.java

```diff
@@ -141,17 +141,17 @@ private static Stream<Arguments> provideCommandStrategies() {
                 Arguments.of("loans/123/transactions?command=chargeRefund", HttpMethod.POST, "createTransactionLoanCommandStrategy",
```

Comment:
Can you please help me find the test you have added to cover the "BatchAPI - status code mismatch in case of exception" fix?